proto/node/akash/market/v1beta5/bidmsg.proto

```
@@ -45,6 +46,16 @@ message MsgCreateBid {
     (gogoproto.jsontag)      = "resources_offer",
     (gogoproto.moretags)     = "yaml:\"resources_offer\""
   ];
+
+  // reclamation_window is the reclamation window duration the provider offers.
+  // If the order requires reclamation, this must be >= the order's min_window.
```

Buf (1.68.4)

[error] cannot find `gogoproto.nullable` in this scope

(COMPILE)